src/main/java/org/isf/medicalstockward/service/MedicalStockWardIoOperations.java

```diff
@@ -249,6 +249,24 @@ public List<MedicalWard> getMedicalsWard(char wardId, boolean stripeEmpty) throw
 		return medicalWards;
 	}
 
+	/**
+	 * Gets all the {@link Medical}s associated to Specified {@link Ward}.
+	 * 
+	 * @param wardId the ward id.
+	 * @param medId  the medical id.
+	 * @return the retrieved medicals.
+	 * @throws OHServiceException if an error occurs during the medical retrieving.
+	 */
+	public List<MedicalWard> getMedicalsWard(String wardId, int medId) throws OHServiceException {
+		List<MedicalWard> medicalWards = repository.findAllWhereWardAndMedical(wardId, medId);
+		for (MedicalWard medicalWard : medicalWards) {
+			double qty = medicalWard.getIn_quantity() - medicalWard.getOut_quantity();
+			medicalWard.setQty(qty);
+		}
+		return medicalWards;
+
+	}
+
```

Comment:
Please the javadoc here cannot be the same of the other overloaded method (same name, different parameters) because in this case we are providing the medical id, so it will retrieve one `MedicalWard` but with different lots, if any (and not all `MedicalWard`s with all lots).

How do you manage empty lots from here on?

Please align javadoc also in the BrowserManager class